src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -216,36 +223,40 @@ boot_newcomp_create(spdid_t spdid, struct cos_compinfo *comp_info, boot_comp_fla
 
 	boot_newcomp_sinv_alloc(spdid);
 
-	if (comp_flags) {
-		boot_newschedcomp_cap_init(spdid, ct, pt, cc);
-
-		thd = sl_thd_comp_init(new_comp_cap_info[spdid].defcompinfo, comp_flags);
-		assert(thd);
-		sl_thd_param_set(thd, sched_param_pack(SCHEDP_PRIO, THD_PRIO));
-	}
-
-	if (comp_flags == BOOT_COMP_FLAG_SCHED) {
-		sl_thd_param_set(thd, sched_param_pack(SCHEDP_BUDGET, THD_BUDGET * 1000));
-		sl_thd_param_set(thd, sched_param_pack(SCHEDP_WINDOW, THD_PERIOD * 1000));
-	}
+	if (comp_flags) boot_newschedcomp_cap_init(spdid, ct, pt, cc);
```

Comment:
We want to do this for memory managers as well?  I'd naively expect `if (comp_flags & ...SCHED...)`